src/Components/HomePage/Sections/AboutUs.js

```diff
@@ -44,7 +44,7 @@ export default function AboutUs() {
             <br />
             <div className={styles.AboutUs_rectangle} />
             <a href="/meettheteam" className={styles.AboutUs_knowMore}>
```

Comment:
Change href="/meettheteam" to href="https://girlcodeit.com"

Reply:
@raks924 Please make the suggested changes asap to get your PR merged.

Reply:
yeah...just gimme some time..it'll be done tongight itself

Reply:
hey i have redirected the button to site 
Please check and confirm .
@vaishali614

Reply:
Looks good

Reply:
@raks924 please pull the latest changes to get your PR merged, otherwise I'll have to assign this issue to someone else.

Reply:
M just doin that . gimme an hour. thanks!

Reply:
hey i have pulled the changes ...check if itz visible now?

Reply:
No, not visible

Reply:
@raks924 please try to do this by tonight.